planner/core/logical_plan_builder.go

```diff
@@ -3056,6 +3061,35 @@ func (b *PlanBuilder) buildProjectionForWindow(ctx context.Context, p LogicalPla
 	return proj, propertyItems[:lenPartition], propertyItems[lenPartition:], newArgList, nil
 }
 
+func (b *PlanBuilder) buildArgs4WindowFunc(ctx context.Context, p LogicalPlan, args []ast.ExprNode, aggMap map[*ast.AggregateFuncExpr]int) ([]expression.Expression, error) {
```

Comment:
Could `buildArgs4WindowFunc` also be used in `buildProjectionForWindow`?

Reply:
no. if we use that, we need to change func `buildArgs4WindowFunc ` to add some projection information into the plan. and then you may find the two methods are same. if so, I think it's much better just call same method instead of two methods